src/display/text_layer.js

```diff
@@ -480,6 +480,7 @@ class TextLayer {
     div.style.opacity = 0;
     div.style.lineHeight = 1;
     div.style.fontSize = "1px";
+    div.style.alignSelf = "flex-start";
```

Comment:
Can we instead use position absolute, to make sure that the div is outside of the layout flow?